src/main/java/graphql/execution/Execution.java

```diff
@@ -1,13 +1,7 @@
 package graphql.execution;
 
 
-import graphql.ExecutionInput;
-import graphql.ExecutionResult;
-import graphql.ExecutionResultImpl;
-import graphql.ExperimentalApi;
-import graphql.GraphQLContext;
-import graphql.GraphQLError;
-import graphql.Internal;
+import graphql.*;
```

Comment:
we avoid * imports

Reply:
Reverted! Any objection to checking in `.idea/codeStyles` to avoid IntelliJ adding those automatically?

Reply:
That's a great idea, please do check in a code style file

Reply:
Done [here](https://github.com/graphql-java/graphql-java/pull/3772/commits/6542fe2b3acaa033afcd92c572c5916a65b7d3a7)